basic/exercise2/script2.js

```diff
@@ -1,10 +1,23 @@
 'use strict';
 
-var side1 = 5;
-var side2 = 6;
-var side3 = 7
-var trianglePerimeter = (side1 + side2 + side3) / 2;
-var triangleArea = Math.sqrt(trianglePerimeter * ((trianglePerimeter - side1) * (trianglePerimeter - side2) * (trianglePerimeter - side3)));
-document.getElementById("triangle").innerHTML = triangleArea;
+function findingTriangleArea(side1, side2, side3) {
+    if (!(side1 > 0)) {
```

Comment:
Please note that you have some repetitive code - with this checking if side is less then 0 - maybe it is a good place to use a function here? You mayb call it validateSide. Function will throw an Error if side is not correct. You may call it function 3 times per each side.